src/app.ts

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+
+import trainingRoutes from './routes/training';
+
+const app = express();
+const port = 8080;
+app.use(trainingRoutes);
+
+app.listen(port);
```

Comment:
You should add ack information that server is running on specified port.

Reply:
.